ClientApp/src/components/UserPage.jsx

```diff
@@ -76,8 +77,26 @@ export class UserPage extends React.Component {
                    id: toUser.id
                },
                message: this.state.currentTextMessage
-            })
+            }),
+       })
+        .then(handleResponse)
+        .catch((error) => {
+           alert(error);
         });
+
+        fetch('api/user/' + currentUser.id, {
```

Comment:
Maybe it's better to nest this fetch request inside a then follow-up when we successfully submit the gem? If we fail to submit the gem, then this request till pulls through.

Reply:
okay looks good